sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorTraceExporter.java

```diff
@@ -62,6 +66,19 @@ public CompletableResultCode export(Collection<SpanData> spans) {
         return telemetryItemExporter.send(telemetryItems);
     }
 
+    private void addInstrumentationToStatBeat(SpanData span) {
+        if (span.getInstrumentationScopeInfo() == null || span.getInstrumentationScopeInfo().getName() == null) {
+            return;
+        }
+        String instrumentationScopeName = span.getInstrumentationScopeInfo().getName();
+        boolean isAnAzureLibraryInstrumentation = instrumentationScopeName.startsWith(
+            "azure-");
+        if (isAnAzureLibraryInstrumentation) {
+            instrumentationScopeName = AZURE_OPENTELEMETRY;
+        }
+        statsbeatModule.getInstrumentationStatsbeat().addInstrumentation(instrumentationScopeName);
+    }
```

Comment:
this is not related to GraalVM, right? i think we need to discuss this first.

Reply:
With GraalVM native, there are 2 main options today: use [this Spring Boot-related project](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring/spring-cloud-azure-starter-monitor) or the standalone exporter with instrumentation libraries or Quarkus.